easybuild/easyconfigs/t/tmux/tmux-2.3-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'tmux'
+version = '2.3'
+
+homepage = 'http://tmux.sourceforge.net/'
+description = """tmux is a terminal multiplexer. It lets you switch easily between
+several programs in one terminal, detach them (they keep running in the background)
+and reattach them to a different terminal."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+toolchainopts = {'optarch': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['https://github.com/tmux/tmux/releases/download/%(version)s/']
```

Comment:
@edmondac existing easyconfigs shouldn't be changed in a way that could impact other stuff, it's possible `tmux-2.2-GCCcore-4.9.3.eb` is used as a dep for something else for example, changing the toolchain would break that (although I admit is unlikely in this case)